src/main/g8/build.sbt

```diff
@@ -1,18 +1,16 @@
-lazy val akkaHttpVersion = "$akka_http_version$"
-lazy val akkaVersion    = "$akka_version$"
+lazy val akkaHttpVersion = "10.0.9"
+lazy val akkaVersion    = "2.5.3"
```

Comment:
When this project was incepted as a seed, the variables for Akka, Akka HTTP and Scala versions, served the purpose of letting the user create quickly a project with the dependencies they considered appropriate.
I personally liked this feature.